gulpfile.babel.js

```diff
@@ -60,7 +61,31 @@ gulp.task('webpack', (cb) => {
   });
 });
 
-gulp.task('serve', () => {
+    
+ 
+// Setting up the test task 
+gulp.task('protractor', ['serve'], function(callback) {
+    gulp
+        .src(['example_spec.js'])
+        .pipe(gulpProtractorAngular({
+            'configFile': 'protractor.conf.js',
+            'debug': false,
+            'autoStartStopServer': true
+        }))
+        .on('error', function(e) {
+            console.log(e);
+        })
+        .on('end', callback);
+});
+
+
+gulp.task('e2e', ['protractor'], function() {
+  process.exit();
```

Comment:
This is maybe not a good idea, but we could store server instance in gulpfile scope and then just stop it. But i thinkg this is better than `process.exit`.